### PR TITLE
Better EN wording + preparing for being translated

### PR DESCRIPTION
**Pull request by svub**

### CODE REVIEW

File: settings.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 ];
 $woo_nimiq_price_service_default = 'coingecko';
 if ( in_array( get_option( 'woocommerce_currency' ), [ 'EUR', 'USD' ] ) ) {
-    $woo_nimiq_price_services['fastspot'] = 'Fastspot (also estimates fees)';
+    $woo_nimiq_price_services['fastspot'] = __( 'Fastspot (also estimates fees)', 'wc-gateway-nimiq' );
```
Comment (sisou) [inline settings.php]:
```suggestion
    $woo_nimiq_price_services['fastspot'] = 'Fastspot' . ' ' . __( '(also estimates fees)', 'wc-gateway-nimiq' );
```

File: settings.php [hunk 2]
```diff
@@ -81,17 +81,17 @@
         'default'     => 'nimiq_watch',
         'options'     => [
             // List available validation services here. The option value must match the file name.
-            'nimiq_watch'  => 'NIMIQ.WATCH (testnet & mainnet)',
-            'json_rpc_nim' => 'Nimiq JSON-RPC API',
-            'nimiqx'       => 'NimiqX (mainnet)',
+            'nimiq_watch'  => 'NIMIQ.WATCH (Testnet & Mainnet)',
+            'json_rpc_nim' => 'Nimiq JSON-RPC API (Testnet & Mainnet)',
```
Comment (sisou) [inline settings.php]:
```suggestion
            'json_rpc_nim' => 'Nimiq JSON-RPC API (Network depends on Nimiq node)',
```

File: validation_services/blockstream.php [hunk 3]
```diff
@@ -45,7 +45,7 @@ public function blockchain_height() {
         }
 
         if ( empty( $latest_block ) ) {
-            return new WP_Error( 'service', __( 'Could not get the current blockchain height from Blockstream.', 'wc-gateway-nimiq' ) );
+            return new WP_Error( 'service', sprintf( __( 'Could not get the current blockchain height from %s.', 'wc-gateway-nimiq' ), 'Blockstream'),
```
Comment (sisou) [inline validation_services/blockstream.php]: Must be a semi-colon at the end, not a comma.

File: validation_services/blockstream.php [hunk 4]
```diff
@@ -172,7 +172,7 @@ public function transaction_found() {
      */
     public function error() {
         if ( empty( $this->transaction ) ) {
-            return __( 'Could not retrieve transaction information from Blockstream.', 'wc-gateway-nimiq' );
+            return sprintf( __( 'Could not retrieve transaction information from %s.', 'wc-gateway-nimiq' ), 'Blockstream'),
```
Comment (sisou) [inline validation_services/blockstream.php]: Must be a semi-colon at the end, not a comma.

File: validation_services/etherscan.php [hunk 5]
```diff
@@ -139,7 +139,7 @@ public function transaction_found() {
      */
     public function error() {
         if ( empty( $this->transaction ) ) {
-            return __( 'Could not retrieve transaction information from Etherscan.', 'wc-gateway-nimiq' );
+            return sprintf( __( 'Could not retrieve transaction information from %ss', 'wc-gateway-nimiq' ), 'Etherscan'),
```
Comment (sisou) [inline validation_services/etherscan.php]: Must be a semi-colon at the end, not a comma.
Also, double `s`:
```suggestion
            return sprintf( __( 'Could not retrieve transaction information from %s', 'wc-gateway-nimiq' ), 'Etherscan'),
```

**Review by sisou — CHANGES_REQUESTED** [review]

File: settings.php (hunk 1 above)
Comment (svub) [reply settings.php]: why `. ' ' . `?

File: validation_services/blockstream.php (hunk 4 above)
Comment (svub) [reply validation_services/blockstream.php]: the dark sides of the copy&edit approach revealed. :blush:

File: settings.php (hunk 1 above)
Comment (sisou) [reply settings.php]: I would remove the service name from the translatable string.

**Review by sisou — APPROVED** [review]

**Pull request merged**